### PR TITLE
New package: IDRIX.VeraCrypt version 1.26.7

### PR DESCRIPTION
**Pull request by ypurpl**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136114)

### CODE REVIEW

**Comment by vvirtues** [comment]
I need some help properly implementing the `Argument` section

**Comment by vvirtues** [comment]
OK it finally works

**Comment by vvirtues** [comment]
Should I add the `ElevationRequirement` property or no? seems to cause problems with validation

**Comment by vvirtues** [comment]
The program does not uninstall properly, will remain draft

**Comment by Trenly** [comment]
> I need some help properly implementing the `Argument` section According to the [validation bot](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=535223&view=logs&j=dc5484e7-078c-5e69-1cbb-0f132d85a80e&t=3990e129-bab1-55f1-b6c9-010c1219a3a4) it doesn't recognize it

Correct. The Microsoft team is working on getting a solution in place to allow the `Agreements` field in manifests. Until this field is available, VeraCrypt won't be able to be added to the repo (unless they change the installer to no longer require `ACCEPTLICENSE=YES`)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-25T16:47:51.3821263Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[ElevationRequirement] Description: Failed to match against any enum values. Error context: <root> Description: Failed to validate against schema associated with property name 'ElevationRequirement'. ] File: IDRIX.VeraCrypt.installer.yaml Manifest Error: Field usage requires verified publishers. [Agreements]

(Automated response - build 675.)

File: manifests/i/IDRIX/VeraCrypt/1.26.7/IDRIX.VeraCrypt.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+PackageIdentifier: IDRIX.VeraCrypt
+PackageVersion: 1.26.7
+InstallerLocale: en-US
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  Silent: /qn ACCEPTLICENSE=YES
+  SilentWithProgress: /qb ACCEPTLICENSE=YES
+ElevationRequirement: ElevatesSelf
```
Comment (stephengillie) [inline manifests/i/IDRIX/VeraCrypt/1.26.7/IDRIX.VeraCrypt.installer.yaml:12]:
```suggestion
ElevationRequirement: elevatesSelf
```
I think that while the property name has to be PascalCase, the value has to be camelCase.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> 2024-01-29T21:50:17.8285962Z ##[error] Manifest Error: Field usage requires verified publishers. [Agreements]

(Automated response - build 679.)

**Comment by Trenly** [comment]
The older version has been restored to the repo with the proper agreements. You should be able to use Komac to submit a new PR for this version, which should automatically pull in the agreements from the version in the repo

**Pull request closed without merging**